File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/MemberAddressEntity.java
package com.yoopoon.market.domain;

import java.io.Serializable;

public class MemberAddressEntity implements Serializable {

	/**
	 * @fieldName: serialVersionUID
	 * @fieldType: long
	 * @Description: TODO
	 */
	private static final long serialVersionUID = 1L;
	public int Id;
	public int UserId;
	public int Member;
	public String Address;
	public String Zip;
	public String Linkman;
	public String Tel;
	public String Adduser;
	public String Addtime;
	public int Upduser;
	public String Updtime;
	public String AddtimeString;
	public String UpdtimeString;
	public int AreaId = 123;

	@Override
	public String toString() {
		return "MemberAddressEntity [Id=" + Id + ", Member=" + UserId + ", Address=" + Address + ", Zip=" + Zip
				+ ", Linkman=" + Linkman + ", Tel=" + Tel + ", Adduser=" + Adduser + ", Addtime=" + Addtime
				+ ", Upduser=" + Upduser + ", Updtime=" + Updtime + ", AddtimeString=" + AddtimeString
				+ ", UpdtimeString=" + UpdtimeString + ", AreaId=" + AreaId + "]";
	}

}

File: Dev/Code/Community.Entity/Model/MemberAddress/MemberAddressEntity.cs
using System;
using Community.Entity.Model.Member;
using YooPoon.Core.Data;
using Community.Entity.Model.Area;

namespace Community.Entity.Model.MemberAddress
{
	public class MemberAddressEntity : IBaseEntity
	{
		/// <summary>
		/// 行政区划ID
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 地址ID
		/// </summary>
		public virtual MemberEntity Member { get; set; }


        /// <summary>
        /// 行政区域
        /// </summary>
        public virtual AreaEntity Area { get; set; }
		/// <summary>
		/// 地址
		/// </summary>
		public virtual string Address { get; set; }
		/// <summary>
		/// 邮编
		/// </summary>
		public virtual string Zip { get; set; }
		/// <summary>
		/// 联系人
		/// </summary>
		public virtual string Linkman { get; set; }
		/// <summary>
		/// 联系电话
		/// </summary>
		public virtual string Tel { get; set; }
		/// <summary>
		/// AddUser
		/// </summary>
		public virtual int Adduser { get; set; }
		/// <summary>
		/// AddTime
		/// </summary>
		public virtual DateTime Addtime { get; set; }
		/// <summary>
		/// UpdUser
		/// </summary>
		public virtual int Upduser { get; set; }
		/// <summary>
		/// UpdTime
		/// </summary>
		public virtual DateTime Updtime { get; set; }
	}
}